certchain/certmanager/helpers_test.go

```diff
@@ -112,6 +112,11 @@ func (c *stubCache) Read(digest string) (*certchain.AugmentedChain, error) {
 	return c.chainMap[digest], nil
 }
 
+func (c *stubCache) ReadLatest() (*certchain.AugmentedChain, error) {
+	// TODO(banaag): unsupported now, will support with in-memory cache PR.
```

Comment:
Nit: TODO is "to do", so I think it's better to start with a verb:

```go
// TODO(banaag): Add support once we have in-memory cache.
```

or something like this.

Reply:
Done